src/Tests/dotnet.Tests/CommandTests/ToolInstallLocalCommandTests.cs

```diff
@@ -316,6 +322,122 @@ public void WhenRunWithPrereleaseAndPackageVersionItShouldSucceed()
             _fileSystem.File.Exists(restoredCommand.Executable.Value);
         }
 
+        [Fact]
+        public void GivenFeedVersionIsTheSameWhenRunWithPackageIdItShouldShowDifferentSuccessMessage()
+        {
+            GetDefaultTestToolInstallLocalCommand().Execute().Should().Be(0);
+            _reporter.Clear();
+            GetDefaultTestToolInstallLocalCommand().Execute().Should().Be(0);
+
+            AssertUpdateSuccess(packageVersion: _packageVersionA);
+            _reporter.Lines.Single()
+                .Should().Contain(
+                    string.Format(
+                        LocalizableStrings.UpdateLocaToolSucceededVersionNoChange,
+                        _packageIdA,
+                        _packageVersionA.ToNormalizedString(),
+                        _manifestFilePath));
+        }
+
+        [Fact]
+        public void GivenFeedVersionIsLowerRunPackageIdItShouldThrow()
+        {
+            GetDefaultTestToolInstallLocalCommand().Execute().Should().Be(0);
+            _mockFeed.Packages.Single().Version = "0.9.0";
+
+            _reporter.Clear();
+            Action a = () => GetDefaultTestToolInstallLocalCommand().Execute();
+            a.Should().Throw<GracefulException>().And.Message.Should().Contain(string.Format(
+                LocalizableStrings.UpdateToLowerVersion,
+                "0.9.0",
+                _packageVersionA.ToNormalizedString(),
+                _manifestFilePath));
+        }
+
+        [Fact]
+        public void GivenFeedVersionIsLowerWithAllowDowngradeOptionRunPackageIdItShouldUpdateToLowerVersion()
```

Comment:
Similar to the previous comment, I don't think in this case we should downgrade unless you explicitly specified the version to downgrade to.